scripts/india/geo/districts.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pandas as pd
+from india.geo.states import IndiaStatesMapper
+
+__author__ = ["Thejesh GN (i@thejeshgn.com)"]
+
+DISTRICTS_MAPPING_CSV = os.path.join(os.path.dirname(__file__),
+                                     "LocalGovernmentDirectory_Districts.csv")
```

Comment:
nit: might be useful to document the schema for this (in https://github.com/datacommonsorg/data/tree/master/scripts/india_lgd/local_government_directory_districts perhaps)... for instance I was wondering what closestDistrictLabel was?

Reply:
- Updated readme
- Used districtLabel intead of closestDistrictLabel to reduce confusion